ipa-core/src/net/test.rs

```diff
@@ -305,6 +305,19 @@ impl TestConfig {
         }
     }
 
+    /// Returns full set of clients to talk to each individual shard in the MPC.
+    #[must_use]
+    pub fn shard_clients(&self) -> [Vec<IpaHttpClient<Shard>>; 3] {
```

Comment:
You won't be able to set ClientIdentity from here. Is this a good place for this method?

I think it would be better to expose something you can iterate over. Something like:
```
pub fn shard_network_confs(&self) -> [NetworkConfig<Shard>; 3] {
...
}
```

Then on the tests `test_make_helpers` you can create the shard configs in a similar pattern to MPC:
```
let shard_clients = conf.shard_network_confs().map(|conf| {
IpaHttpClient::shards_from_conf(
                &IpaRuntime::current(),
                &conf,
                &ClientIdentity::None,
            )
})
```
Gives the test the option to set the settings as needed like MPC.

Reply:
yea, it is no longer relevant as we don't need shard clients anymore